tests/handlers/test_receipts.py

```diff
@@ -331,6 +331,27 @@ def test_leaves_our_private_and_their_public(self):
             ],
         )
 
+    def test_we_do_not_mutate(self):
+        events = [
+            {
+                "content": {
+                    "$1435641916114394fHBLK:matrix.org": {
+                        ReceiptTypes.READ_PRIVATE: {
+                            "@rikj:jki.re": {
+                                "ts": 1436451550453,
+                            }
+                        }
+                    }
+                },
+                "room_id": "!jEsUZKDJdhlrceRyVU:example.org",
+                "type": "m.receipt",
+            }
+        ]
+        events_copy = events.copy()
```

Comment:
I realized that this wasn't a `deepcopy` so wasn't testing what we thought!

When I modified it, well...the code was mutating things. 😢